modules/seqexec/web/client/src/webpack/webpack.parts.js

```diff
@@ -57,11 +57,25 @@ module.exports.lessLoader = (options = {}) => {
 };
 
 // Extract css to a file, use only in production
-module.exports.extractCSS = ({ devMode, include, exclude, use = [] }) => {
+module.exports.extractCSS = ({
+  devMode,
+  include,
+  exclude,
+  use = [],
+  ci = false
+}) => {
   // Output extracted CSS to a file
   const plugin = new MiniCssExtractPlugin({
-    filename: devMode ? "[name].css" : "[name].[contenthash].css",
-    chunkFilename: devMode ? "[id].css" : "[id].[contenthash].css"
+    filename: devMode
+      ? "[name].css"
+      : ci
+        ? "[name].css"
+        : "[name].[contenthash].css",
+    chunkFilename: devMode
+      ? "[id].css"
+      : ci
+        ? "[id].css"
+        : "[id].[contenthash].css"
```

Comment:
This is rather confusing. I can see what you are doing, but the nested ? : without parentheses makes me feel uncomfortable.

Reply:
This is made by prettier a very standard js beautifier. I'll try to get something more readable in